internal/entitlement/httpdriver/parser.go

```diff
@@ -78,7 +78,7 @@ func (parser) ToBoolean(e *entitlement.Entitlement) (*api.EntitlementBoolean, er
 		DeletedAt:  boolean.DeletedAt,
 		FeatureId:  boolean.FeatureID,
 		Id:         &boolean.ID,
-		Metadata:   nil, // implement
+		Metadata:   &boolean.Metadata,
```

Comment:
If this is nil, we should set this field to nil. e.g. you can create something in convert for that.